### PR TITLE
[#199] Drop support for Pivotal Tracker

### PR DESCRIPTION
**Pull request by klappradla**
This removes Pivotal Tracker from the list of supported ticket systems as described in #199:
- remove the adapter and related tests
- adjust the docs to no longer list pivotal tracker

While updating the list of supported trackers, I also added a small note to encourage people to implement support for Pivotal by themselves if they need it 🤷‍♂ Let's see.

Up next would be: draft a release, release it and adjust the descriptions in the respective places (Chrome / Firefox).

It would also be neat if someone with Github superpowers could remove the `pivotal-tracker` tag from the description of this repo 😉

### CODE REVIEW

File: src/common/search.js [hunk 1]
```diff
@@ -17,10 +16,13 @@ async function attempt(scan, loc, doc) {
 }
 
 function aggregate(results) {
-  return results.reduce(({ tickets, errors }, result) => ({
-    errors: errors.concat(result.error ? [result.error] : []),
-    tickets: tickets.concat(result.tickets || []),
-  }), { tickets: [], errors: [] });
+  return results.reduce(
```
Comment (klappradla) [inline src/common/search.js]: Note: this is only a side-effect of the linter having fun after line 32 got shorter 😉

File: src/common/adapters/pivotal.js [hunk 2]
```diff
@@ -1,55 +0,0 @@
-import {
-  $all,
-  $classed,
```
Comment (juffel) [inline src/common/adapters/pivotal.js]: Looks like we could now even throw out some functions from `src/common/adapters/helpers.js` like `classed` and `closest` 🗑

File: README.md [hunk 3]
```diff
@@ -33,9 +33,10 @@ Currently, we support:
 - [GitLab](https://gitlab.com/)
 - [Jira](https://www.atlassian.com/software/jira)
 - [Ora](https://ora.pm/)
-- [Pivotal Tracker](https://www.pivotaltracker.com/)
 - [Trello](https://trello.com/)
 
+Your ticket system is missing? Go ahead and implement an adapter for it! We are always happy about contributions and here to support you 👋.
```
Comment (pmeinhardt) [inline README.md]: 😍

Comment (juffel) [inline README.md]: 👍

File: src/common/adapters/pivotal.js (hunk 2 above)
Comment (klappradla) [reply src/common/adapters/pivotal.js]: Neat! I checked which ones were not used and removed them ✅

**Comment by pmeinhardt** [comment]
Awesome @klappradla. Good to be merged from my point of view 🙂

**Comment by klappradla** [comment]
It was quite some time since the last release @pmeinhardt 🤔 

I'll release a new version, ok?

**Comment by pmeinhardt** [comment]
I think the "summary" button might not quite work as before (since we have the full commit messages now, not just the title). I don't think a lot of people are using that anyway though and we can address that later. 🤔

**Pull request merged**